packages/royalties/contracts/MultiRoyaltyDistributer.sol

```diff
@@ -0,0 +1,166 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.0;
+
+import {ERC165Upgradeable} from "@openzeppelin/contracts-upgradeable/utils/introspection/ERC165Upgradeable.sol";
+import {EnumerableSet} from "@openzeppelin/contracts/utils/structs/EnumerableSet.sol";
+import {Clones} from "@openzeppelin/contracts/proxy/Clones.sol";
+
+import {
+    IEIP2981MultiReceiverRoyaltyOverride
+} from "@manifoldxyz/royalty-registry-solidity/contracts/overrides/IMultiReceiverRoyaltyOverride.sol";
+import {IMultiRoyaltyDistributer} from "./interfaces/IMultiRoyaltyDistributer.sol";
+import {
+    IRoyaltySplitter,
+    IERC165
+} from "@manifoldxyz/royalty-registry-solidity/contracts/overrides/IRoyaltySplitter.sol";
+import {IEIP2981} from "@manifoldxyz/royalty-registry-solidity/contracts/specs/IEIP2981.sol";
+import {IRoyaltyManager, Recipient} from "./interfaces/IRoyaltyManager.sol";
+
+/// @title MultiRoyaltyDistributer
+/// @author The sandbox
+/// @dev import for Token contracts EIP3981 Royalty distribution and split for sandbox and the creator using splitters.
```

Comment:
Please fix casing / typos (wrong EIP reference). Comments are messy and need to be tidied for readability.
Import for Token contracts EIP2981 Royalty distribution and split for The Sandbox and the creator using splitters.